{"name": "test_invalid_login", "status": "failed", "statusDetails": {"message": "AssertionError: Expected error message not shown\nassert '' != ''", "trace": "driver = <appium.webdriver.webdriver.WebDriver (session=\"3e48dc5dca8877d8eb29c72940beb8443c0e2e44\")>\n\n    def test_invalid_login(driver):\n        lp = LoginPage(driver)\n        # locked_out_user always fails\n        lp.login(\"locked_out_user\", \"secret_sauce\")\n        err = lp.get_error()\n>       assert err != \"\", \"Expected error message not shown\"\nE       AssertionError: Expected error message not shown\nE       assert '' != ''\n\ntests\\test_login.py:18: AssertionError"}, "attachments": [{"name": "screenshot-android", "source": "67de51d0-3813-4c36-bdbc-efa0c0c757ed-attachment.png", "type": "image/png"}], "start": 1757841343491, "stop": 1757841362178, "uuid": "09526863-3d5d-4b47-9391-b17f4b3bde4b", "historyId": "6485cd1aa206d185bd707f422a9a9238", "testCaseId": "6485cd1aa206d185bd707f422a9a9238", "fullName": "tests.test_login#test_invalid_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "DB-L-174"}, {"name": "thread", "value": "14392-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}], "titlePath": ["tests", "test_login.py"]}